src/Arcus.Observability.Tests.Unit/Serilog/ApplicationInsightsTelemetryConverterTests.cs

```diff
@@ -122,6 +122,53 @@ public void LogDependency_WithDependency_CreatesDependencyTelemetry()
             });
         }
 
+        [Fact]
+        public void LogServiceBusDependency_WithServiceBusDependency_CreatesDependencyTelemetry()
```

Comment:
Can we add tests for our methods who are specific for queue & topic please? One overload is ok

Reply:
Done that.